GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -1068,10 +1080,11 @@ void AddArtificialRevisions(ObjectId filteredCurrentCheckout)
                         CommitUnixTime = 0,
                         CommitterEmail = userEmail,
                         Subject = ResourceManager.TranslatedStrings.Index,
-                        ParentIds = new[] { filteredCurrentCheckout },
+                        ParentIds = new[] { CurrentCheckout },
                         HasNotes = true
                     };
-                    _gridView.Add(indexRev);
+
+                    _gridView.Add(indexRev, addParentGraph: addParentGraph);
```

Comment:
```suggestion
                    _gridView.Add(indexRev, addParentGraph);
```

Reply:
Gerhard does this only if necessary, namely if default arguments are in the play.